### PR TITLE
add function update kyber network on feehandler via proxy

### PR DESCRIPTION
**Pull request by ductm54**

### CODE REVIEW

File: contracts/sol6/Dao/KyberFeeHandler.sol [hunk 1]
```diff
@@ -326,6 +332,14 @@ contract KyberFeeHandler is IKyberFeeHandler, Utils5, BurnConfigPermission {
         daoSetter = address(0);
     }
 
+    /// @dev update kyber network contract address via kyber proxy.
+    function updateNetworkContract() external {
+        address _kyberNetwork = networkProxy.kyberNetwork();
```
Comment (ilanDoron) [inline contracts/sol6/Dao/KyberFeeHandler.sol:337]: this should be 

onlyBurnConfigSetter

File: test/sol6/kyberFeeHandler.js [hunk 2]
```diff
@@ -1631,11 +1633,31 @@ contract('KyberFeeHandler', function(accounts) {
             });
         });
     });
+
+    it("test upgradeable", async() => {
```
Comment (ilanDoron) [inline test/sol6/kyberFeeHandler.js:1637]: please add to the name

update network address

File: contracts/sol6/Dao/KyberFeeHandler.sol [hunk 3]
```diff
@@ -31,6 +32,11 @@ import "../utils/zeppelin/SafeMath.sol";
  *      2. Network Fee distribtuion. per epoch Kyber fee Handler reads current distribution from Kyber DAO.
```
Comment (manhlx3006) [inline contracts/sol6/Dao/KyberFeeHandler.sol:32]: pls help to fix this typo: "distribtuion"

File: test/sol6/kyberFeeHandler.js [hunk 4]
```diff
@@ -1631,11 +1633,31 @@ contract('KyberFeeHandler', function(accounts) {
             });
         });
     });
+
+    it("test upgradeable - update network address", async() => {
+        admin = accounts[9];
+        tempNetwork = await KyberNetworkProxy.new(admin);
+        tempProxy = await KyberNetworkProxy.new(admin);
+        tempFeeHandler = await FeeHandler.new(daoSetter, tempProxy.address, kyberNetwork, knc.address, BURN_BLOCK_INTERVAL, burnConfigSetter);
```
Comment (manhlx3006) [inline test/sol6/kyberFeeHandler.js:1641]: `tempNetwork` should be Network instance (or just normall addr to test)?

Comment (ductm54) [reply test/sol6/kyberFeeHandler.js:1641]: fixed. change to network instance

File: contracts/sol6/Dao/KyberFeeHandler.sol (hunk 3 above)
Comment (ductm54) [reply contracts/sol6/Dao/KyberFeeHandler.sol:32]: fixed

**Pull request merged**